README.md

```diff
@@ -3,12 +3,12 @@ Arentas, Martinas, Pijus
 
 # Setup
 1. Open solution file (karma.sln) with visual studio
-1. Build the project
```

Comment:
this was done on purpose (it is markdown syntax)
https://www.markdownguide.org/basic-syntax/#ordered-lists